src/univalue/test/test_json.cpp

```diff
@@ -1,26 +0,0 @@
-// Test program that can be called by the JSON test suite at
-// https://github.com/nst/JSONTestSuite.
```

Comment:
Not sure if we should remove this. Developers can still use it locally or it can be added to CI later?

Reply:
I doubt any developer is ever going to use this locally. It could be added to the CI (and looks like it at least partly was without realising this code is never compiled), but I'm not sure it's worth it.